### PR TITLE
Updating ConvertJSON to handle non-openconfig yang model for opstate

### PR DESCRIPTION
**Pull request by SeanCondon**

### CODE REVIEW

File: pkg/modelregistry/jsonvalues/convertjson.go [hunk 1]
```diff
@@ -109,9 +109,9 @@ func CorrectJSONPaths(jsonBase string, jsonPathValues []*change.ConfigValue,
 				break
 
 			} else if hasPrefixMultipleIdx(jsonPathWildIndex, modelPathWildIndex) {
-				jsonSubPath := jsonPathWildIndex[len(modelPathWildIndex):]
+				//jsonSubPath := jsonPathWildIndex[len(modelPathWildIndex):]
```
Comment (Andrea-Campanella) [inline pkg/modelregistry/jsonvalues/convertjson.go]: why was this commented out ? if it's not needed please remove

File: pkg/modelregistry/jsonvalues/convertjson.go [hunk 2]
```diff
@@ -109,9 +109,9 @@ func CorrectJSONPaths(jsonBase string, jsonPathValues []*change.ConfigValue,
...
 
-				newTypeValue, err := assignModelType(paths, modelPath, jsonSubPath, jsonPathValue)
+				newTypeValue, err := assignModelType(paths, modelPath, jsonPathValue)
```
Comment (Andrea-Campanella) [inline pkg/modelregistry/jsonvalues/convertjson.go]: I see no difference in the usage of the method from before but the following lines are different, is it worth consolidating by extracting this line and the error evaluation or am I missing some logic.

File: pkg/modelregistry/jsonvalues/convertjson.go (hunk 1 above)
Comment (SeanCondon) [reply pkg/modelregistry/jsonvalues/convertjson.go]: removed

File: pkg/modelregistry/jsonvalues/convertjson.go (hunk 2 above)
Comment (SeanCondon) [reply pkg/modelregistry/jsonvalues/convertjson.go]: updated

**Pull request merged**